test/e2e/indexTable/indexTable.spec.ts

```diff
@@ -24,26 +24,65 @@ test('IndexTable - bulk action', async ({ page }) => {
   const indexTablePage = new IndexTablePage(page);
   await indexTablePage.goto();
 
+  let isCheckboxCorrect: boolean, tempText: string | null;
+  isCheckboxCorrect = await indexTablePage.verifyCheckboxValues(
+    new Array(5).fill(false)
+  );
+  expect(isCheckboxCorrect).toBe(true);
+
   await indexTablePage.checkboxes.first().click();
```

Comment:
can you create a separate locator for the first checkbox and name it as `selectAllCheckbox` maybe so we understand what the first checkbox is. can you also assert for when it's `indeterminate` state other than the `checked` and `unchecked` states. we need this since we removed the screenshots and this is the only checkbox that has this state in this component.

Reply:
yes! what I am not sure is how to check for `indeterminate` state... when I saw the checkbox by inspecting (and also see the accessibility tab), it seems the checkbox can only be checked or not checked, not sure if there are some specific properties that I should check to consider that as `intedeterminate`

Reply:
can you try something like this: https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-attribute
```ts
// indeterminate checkbox
await expect(indexTablePage. selectAllCheckbox).toHaveAttribute('aria-checked', 'mixed');
```

ref: https://github.com/glints-dev/glints-aries/blob/v4/src/%40next/Checkbox/Checkbox.tsx#L59